### DISCUSSION TITLE
Union Spacer

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#446] [General]
...

**Comment by Kor-Angar** [comment]
Знаешь я его прошлый раз ставил на чистую версию и проблем не было. Потом были проблемы из-за файлов в авторане, но я этот момент учёл. Потом я в нем редактировал конвертированный мир Готики 1 и все было прекрасно. А сейчас попробовал его через воркшоп, мир загружается, но экран чёрный. Видимо я что то ниучел или по забыл :-)

**Reply by Kor-Angar** [reply]
> Если экран черный, значит спейсер не видит ресурсы игры. Все открываемые миры должны быть в Gothic II\_work\Data\Worlds. Попробуй так.

**Reply by Kor-Angar** [reply]
я туда и ложил
![image](https://user-images.githubusercontent.com/49514764/170469779-378f3155-0344-4ec9-93bc-35418dffd4ff.png)

**Reply by dosinabox** [reply]
Спейсер из мастерской?
Попробуй без DX11.

**Reply by Kor-Angar** [reply]
> Спейсер из мастерской?

да. файл должен быть в корне или во вложенной папки?

**Reply by Kor-Angar** [reply]
открылся :-)


![image](https://user-images.githubusercontent.com/49514764/170471399-8f731c87-c72a-4ef7-9838-1874a3680637.png)

**Reply by dosinabox** [reply]
Файл мира?
_work\Data\Worlds\NEWWORLD. ZEN
или
_work\Data\Worlds\NEWWORLD\NEWWORLD. ZEN

**Comment by Kor-Angar** [comment]
> А он не должен быть читабельным. Чтобы стал читабельным, нужно сохранить его как uncompiled в ascii. В такой файл сохраняться только объекты с координатами, без геометрии мира и текстур. После этого его можно будет открыть только в блокноте или компилировать в спейсере с мешами геометрии 3ds.

**Reply by Kor-Angar** [reply]
Для Г1 не может, а вот для г2 НВ он может быть читабильный. Сначало идёт компилированнвй код мира, а потом под читабильный текст. И можно править читабильный текст, и эти правки при загрузки мира игрой будут учитываться. Таким образом я правил файл Готику 1, Сначало один объект через Спейсер юнион, а потом все остальные через блокнот.

**Reply by Kor-Angar** [reply]
в итоге я просто потом вставлял под скомпилированный мир, некомпилированный код
![image](https://user-images.githubusercontent.com/49514764/170469446-57570810-7078-4085-8938-ff27a95d22eb.png)

**Reply by Kor-Angar** [reply]
правки в zen мире через блокнотик работает :-) Union Spaser тоже корректно сохраняет такие файлы

![image](https://user-images.githubusercontent.com/49514764/170488069-a0b19c36-a3e0-4df7-9b38-4ee3e12a6262.png)
